eclair-core/src/main/scala/fr/acinq/eclair/blockchain/fee/FeeProvider.scala

```diff
@@ -95,6 +97,9 @@ object FeeratePerKw {
    * hence feerate-per-kw >= 253
    *
    * See also https://github.com/ElementsProject/lightning/pull/1251
+   * Notes:
+   *  - The size of a block is limited to 4,000,000 weight units which corresponds to 1,000,000 virtual bytes.
+   *  - The transaction fee doesn't depend on the amount being transferred. It depends on the amount of data being transferred.
```

Comment:
This is basic bitcoin trivia, and this repository isn't meant to be a learning resource for bitcoin, please remove it.

Reply:
OK. Removed.